env.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+)
+
+const (
```

Comment:
Can you add a TODO_DOCUMENT somewhere here please

Reply:
How about I just document it right now? :)

Reply:
Done.